tests/kafkatest/services/mirror_maker.py

```diff
@@ -108,11 +108,18 @@ def __init__(self, context, num_nodes, source, target, whitelist=None, blacklist
             raise Exception("offsets_storage should be 'kafka' or 'zookeeper'. Instead found %s" % self.offsets_storage)
 
         self.offset_commit_interval_ms = offset_commit_interval_ms
+        self.interceptor_classes = interceptor_classes
```

Comment:
Are these for producer or consumer? In MM you could potentially have both. It would be good to avoid ambiguity here, even if only one of the two is currently needed.